src/helpers/ResolveRundownIntoPlaylist.ts

```diff
@@ -59,3 +60,17 @@ export function ResolveRundownIntoPlaylist(
 
 	return { resolvedPlaylist, untimedSegments }
 }
+
+function isSegmentTitleReadyOnAir(segment: UnrankedSegment): boolean {
+	return /klar[\s-]*on[\s-]*air/im.test(segment.name)
+}
+
+function isSegmentTitleContinuity(segment: UnrankedSegment): boolean {
+	return /^\s*continuity\s*$/i.test(segment.name)
+}
+
+function isSegmentBackTimeCorrectFormat(segment: UnrankedSegment): boolean {
+	const backTime: string = segment.iNewsStory.fields.backTime ? segment.iNewsStory.fields.backTime : ''
```

Comment:
This syntax differs from https://github.com/tv2/inews-ftp-gateway/blob/a39692166a26de32a3a5c4e77e28df465847cddd/src/classes/RundownManager.ts#L86 in code style, but they both do the same job?